### ISSUE TITLE
Accessibility issue - selected month is not announced by screen reader

### ISSUE THREAD

**Issue by Ke1sy** [Hacker0x01/react-datepicker#3558]
**Describe the bug**
When you change month, it is expected that the new month with year will be announced by screen reader. 
Currently after clicking next/prev month button, user doesn’t know that month has changed and what month is currently selected, because nothing is announced. It is recommended to add aria-live="polite" and announce changed information to user.

**To Reproduce**
1. Start the screen reader and open the date-picker with keyboard.
2. Navigate to previous or next month button, then click it

**Expected behavior**
Selected month with year will be announced by screen reader.

**Desktop (please complete the following information):**

- OS: [e.g. iOS] Windows 10
- Browser: Chrome (v. 100.0.4896.75), Firefox

**Comment by martijnrusschen** [comment]
fixed by https://github.com/Hacker0x01/react-datepicker/pull/3817
